csv_download.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: CSV Download
+nav_order: 6
+has_children: false
+---
+
+# CSV Download
```

Comment:
maybe CSV Bulk Download? or Relational Tables

Reply:
changed to CSV Bulk Download